libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -1291,7 +1213,7 @@ def invoke(
                         step.invoke,
                         input,
                         # mark each step as a child run
-                        patch_config(config, run_manager.get_child()),
+                        patch_config(deepcopy(config), run_manager.get_child()),
```

Comment:
should we use get config list here too? so we don't forget the deepcopies later on?